frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-export default function SuggestedLabels({ card, labels }) {
-    if (labels.length <= 5) {
+export default function SuggestedLabels({ card, labels, onSelectLabel }) {
```

Comment:
React/Redux has a specific style of using actions, reducers, containers, and components to pass, get, and modify state. I would like to see these new changes keeping to those conventions for ease of maintenance. 

A few ways this could happen:
* SuggestedLabels could be a separate component with it's own which is called within DataCard and has an action that fetches the suggested labels
* SuggestedLabels gets turned into an action which is called by DataCard and the render part is folded into DataCard's render function

In general, I'd like to move away from adding extra `.jsx` files to component folders. If a function is part of a component, I'd prefer it went in the component's index file (see other components like History for examples). I see I missed this in an earlier branch as `CardData.jsx` and `SuggestedLabels.jsx` are not new, so apologies if this causes confusion!

Reply:
Ok so I'm going to resolve this as per our discussion with Jason. The takeaways here moving forward:

* functional components can be added as files in component folders if they are subcomponents of that component
* Calls to the api can go in these functional components so long as what they are calling isn't needed elsewhere. If the component is pulling something that is shared, then that should go in redux which can simplify passing out shared data to components